src/AppHosts/SuperMemoAssistant/UI/CollectionSelectionWindow.xaml.cs

```diff
@@ -244,6 +250,61 @@ private void BtnUpdates_Click(object sender, RoutedEventArgs e)
         Close();
     }
 
+    private void ShowQuoteOfTheDay(object sender,
+                                   RoutedEventArgs e)
+    {
+      // Tab separated file with a heading
+      // Quote, Author, Url, Title
+      var quoteFile = SMAFileSystem.AppRootDir.CombineFile("quotes.tsv");
+      if (quoteFile.Exists())
+      {
+        try
+        {
+          var lines = File.ReadAllLines(quoteFile.FullPath);
```

Comment:
That's not going to work with the latest version. You have to use `SMAFileSystem.GetAppExeFilePath` to combine your file's name with the executed app's directory.

`SMAFileSystem.AppRootDir` will return e.g. `C:\Users\Jamesb\AppData\Local\SuperMemoAssistant`
`SMAFileSystem.GetAppExeFilePath` will combine your file with e.g. `C:\Users\Jamesb\AppData\Local\SuperMemoAssistant\app-2.0.4.84`

Reply:
Ok noted.